README.md

```diff
@@ -37,46 +41,45 @@ endpoint permissions in the documentation as well you need to name the function
+| **middlewares**							                    | object		      | 	❌	| Configuration parameters for parsing middlewares.                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
+| **middlewares**.formatter					            | function		    | 	❌	| Custom formatter function for middleware. If not provided, the default one will be used.                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| **middlewares**.middlewareName			         | string		      | 	✅	| Name of the middleware.                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
+| **middlewares**.closure					              | string		      | 	✅	| Name of the middleware closure.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+| **middlewares**.maxParamDepth					              | number		      | 	❌	| Max copying depth of middleware parameter. By default is 5. to Max depth of middleware parameter. Default value is 5.                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
```

Comment:
@kot-1999 this option was not renamed correctly. it should be `Max depth of middleware parameter. Default value is 5.`